doc/supported_file_formats.adoc

```diff
@@ -0,0 +1,101 @@
+= Supported File Formats
+include::../include/config.adoc[]
+:description: The file formats Minetest supports.
+:keywords: fileformats
+
+Minetest is able to read several types of file formats for use as assets in-game, each with varying usecase and support.
+
+== Images
+[grid="all"]
+[options="header",stripes=even,cols="^.^,<.^,^.^,^.^,^.^"]
+|===
+| Format
+^| General Usecase
+| Can be written by Lua
+| Can be read by Lua
+| Lossless optimization
+
+| PNG (`.png`)
+| All-round good losslessly compressed image format.
+| Yes (`minetest.encode_png()` ^(5.5+)^) (Third-party library ^(-5.4)^)
```

Comment:
I suggest to add that `minetest.encode_png()` will can produce rather large RGBA image files.

(When I checked, devtest checkerboard was about 20 times the size that it should have been.)

I also suggest to add a bit regarding OptiPNG usage.

Reply:
Same goes for my implementation.